### ISSUE TITLE
Ability to comment on queue post

### ISSUE THREAD

**Issue by aigagror** [illinois/queue#163]
For CS 225, students may say something like "segfault" or "code does not run". This violates our policy of having topic specific descriptions and proof of nontrivial work put into the issue before going on queue. It would be nice to have the ability for teachers to comment on their queue post to inform them of this. This would avoid the awkwardness of skipping them after telling them in person that they need to update their queue post

**Comment by aigagror** [comment]
I actually think Peter’s suggestion is even better. It would avoid the tendency to make this a chat and still serve its purpose

**Comment by PiotrGalusza** [comment]
I was thinking we'd change their border color and give them some kind of comment/warning. I was mostly thinking that a two-way convo wasn't needed, so we could limit the direction of communication

**Comment by nwalters512** [comment]
Yeah, thinking more about that, that could be a very nice UX. We could allow staff to fill in some template responses with general suggestions for question improvements.

**Comment by PiotrGalusza** [comment]
Yep yep. Templated/generalized responses for common issues, and maybe the final option is to type in a custom message (for unique/unusual cases)
